ng-dialog/ng-dialog.d.ts

```diff
@@ -106,5 +112,9 @@ declare module angular.dialog {
 		 * Scope object that will be passed to dialog. If you use controller with separate $scope service this object will be passed to $scope.$parent param.
 		 */
 		scope?: ng.IScope;
+		/**
+		 * Any serializable data that you want to be stored in the controller's dialog scope.
+		 */
+		data?: string|Object|Array<any>;
```

Comment:
Didn't understood exactly why you closed it. 
In the future all you had to do is simply re-commit to the same branch and nothing else need to be done as the PR will sync automatically.